#171602012 - Ender Yılmaz
#Bilişim Sistemleri Mühendisliği
#Elektronik Komponent Kayıt Kontrol Sistemi
from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_komponentler(object):
    def setupUi(self, komponentler):
        komponentler.setObjectName("komponentler")
        komponentler.resize(623, 279)
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("buton_resim/anakart.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        komponentler.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(komponentler)
        self.centralwidget.setObjectName("centralwidget")
        self.liste = QtWidgets.QTableWidget(self.centralwidget)
        self.liste.setGeometry(QtCore.QRect(290, 10, 321, 211))
        self.liste.setRowCount(5)
        self.liste.setColumnCount(3)
        self.liste.setObjectName("liste")
        self.ekle_btn = QtWidgets.QPushButton(self.centralwidget)
        self.ekle_btn.setEnabled(True)
        self.ekle_btn.setGeometry(QtCore.QRect(180, 10, 101, 41))
        font = QtGui.QFont()
        font.setBold(False)
        font.setWeight(50)
        font.setStyleStrategy(QtGui.QFont.PreferDefault)
        self.ekle_btn.setFont(font)
        self.ekle_btn.setLayoutDirection(QtCore.Qt.LeftToRight)
        self.ekle_btn.setAutoFillBackground(False)
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap("buton_resim/ekle.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.ekle_btn.setIcon(icon1)
        self.ekle_btn.setIconSize(QtCore.QSize(32, 32))
        self.ekle_btn.setObjectName("ekle_btn")
        self.sil_btn = QtWidgets.QPushButton(self.centralwidget)
        self.sil_btn.setGeometry(QtCore.QRect(180, 60, 101, 41))
        icon2 = QtGui.QIcon()
        icon2.addPixmap(QtGui.QPixmap("buton_resim/sil.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.sil_btn.setIcon(icon2)
        self.sil_btn.setIconSize(QtCore.QSize(32, 32))
        self.sil_btn.setObjectName("sil_btn")
        self.guncelle_btn = QtWidgets.QPushButton(self.centralwidget)
        self.guncelle_btn.setGeometry(QtCore.QRect(180, 110, 101, 41))
        icon3 = QtGui.QIcon()
        icon3.addPixmap(QtGui.QPixmap("buton_resim/guncelle.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.guncelle_btn.setIcon(icon3)
        self.guncelle_btn.setIconSize(QtCore.QSize(32, 32))
        self.guncelle_btn.setObjectName("guncelle_btn")
        self.listele_btn = QtWidgets.QPushButton(self.centralwidget)
        self.listele_btn.setGeometry(QtCore.QRect(180, 210, 101, 41))
        icon4 = QtGui.QIcon()
        icon4.addPixmap(QtGui.QPixmap("buton_resim/listele.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.listele_btn.setIcon(icon4)
        self.listele_btn.setIconSize(QtCore.QSize(32, 32))
        self.listele_btn.setObjectName("listele_btn")
        self.tur_label = QtWidgets.QLabel(self.centralwidget)
        self.tur_label.setGeometry(QtCore.QRect(20, 10, 41, 16))
        self.tur_label.setObjectName("tur_label")
        self.deger_label = QtWidgets.QLabel(self.centralwidget)
        self.deger_label.setGeometry(QtCore.QRect(20, 40, 41, 16))
        self.deger_label.setObjectName("deger_label")
        self.stok_label = QtWidgets.QLabel(self.centralwidget)
        self.stok_label.setGeometry(QtCore.QRect(20, 70, 41, 16))
        self.stok_label.setObjectName("stok_label")
        self.deger = QtWidgets.QLineEdit(self.centralwidget)
        self.deger.setGeometry(QtCore.QRect(60, 40, 113, 20))
        self.deger.setObjectName("deger")
        self.stok = QtWidgets.QLineEdit(self.centralwidget)
        self.stok.setGeometry(QtCore.QRect(60, 70, 113, 20))
        self.stok.setObjectName("stok")
        self.tur = QtWidgets.QComboBox(self.centralwidget)
        self.tur.setGeometry(QtCore.QRect(60, 10, 111, 22))
        self.tur.setObjectName("tur")
        self.tur.addItem("")
        self.tur.addItem("")
        self.tur.addItem("")
        self.tur.addItem("")
        self.tur.addItem("")
        self.ara_btn = QtWidgets.QPushButton(self.centralwidget)
        self.ara_btn.setGeometry(QtCore.QRect(560, 230, 51, 21))
        icon5 = QtGui.QIcon()
        icon5.addPixmap(QtGui.QPixmap("buton_resim/arama.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.ara_btn.setIcon(icon5)
        self.ara_btn.setIconSize(QtCore.QSize(16, 16))
        self.ara_btn.setObjectName("ara_btn")
        self.ara = QtWidgets.QLineEdit(self.centralwidget)
        self.ara.setGeometry(QtCore.QRect(290, 230, 261, 21))
        self.ara.setObjectName("ara")
        self.ozel_tur = QtWidgets.QComboBox(self.centralwidget)
        self.ozel_tur.setGeometry(QtCore.QRect(10, 210, 161, 41))
        self.ozel_tur.setObjectName("ozel_tur")
        self.ozel_tur.addItem("")
        self.ozel_tur.addItem("")
        self.ozel_tur.addItem("")
        self.ozel_tur.addItem("")
        self.ozel_tur.addItem("")
        self.ozel_tur.addItem("")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(50, 180, 81, 16))
        font = QtGui.QFont()
        font.setPointSize(12)
        font.setBold(True)
        font.setWeight(75)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.temizle_btn = QtWidgets.QPushButton(self.centralwidget)
        self.temizle_btn.setGeometry(QtCore.QRect(180, 160, 101, 41))
        icon6 = QtGui.QIcon()
        icon6.addPixmap(QtGui.QPixmap("buton_resim/temizle.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.temizle_btn.setIcon(icon6)
        self.temizle_btn.setIconSize(QtCore.QSize(32, 32))
        self.temizle_btn.setObjectName("temizle_btn")
        komponentler.setCentralWidget(self.centralwidget)
        self.statusbar = QtWidgets.QStatusBar(komponentler)
        self.statusbar.setObjectName("statusbar")
        komponentler.setStatusBar(self.statusbar)

        self.retranslateUi(komponentler)
        QtCore.QMetaObject.connectSlotsByName(komponentler)

    def retranslateUi(self, komponentler):
        _translate = QtCore.QCoreApplication.translate
        komponentler.setWindowTitle(_translate("komponentler", "Elektronik Komponentler"))
        self.ekle_btn.setText(_translate("komponentler", "        Ekle"))
        self.sil_btn.setText(_translate("komponentler", "           Sil"))
        self.guncelle_btn.setText(_translate("komponentler", "   Güncelle"))
        self.listele_btn.setText(_translate("komponentler", "     Listele"))
        self.tur_label.setText(_translate("komponentler", "Tür     :"))
        self.deger_label.setText(_translate("komponentler", "Değer :"))
        self.stok_label.setText(_translate("komponentler", "Stok    :"))
        self.tur.setItemText(0, _translate("komponentler", "Direnç"))
        self.tur.setItemText(1, _translate("komponentler", "Diyot"))
        self.tur.setItemText(2, _translate("komponentler", "Bobin"))
        self.tur.setItemText(3, _translate("komponentler", "Transistör"))
        self.tur.setItemText(4, _translate("komponentler", "Kondansatör"))
        self.ara_btn.setText(_translate("komponentler", "Ara"))
        self.ozel_tur.setItemText(0, _translate("komponentler", "Tümü"))
        self.ozel_tur.setItemText(1, _translate("komponentler", "Direnç"))
        self.ozel_tur.setItemText(2, _translate("komponentler", "Diyot"))
        self.ozel_tur.setItemText(3, _translate("komponentler", "Bobin"))
        self.ozel_tur.setItemText(4, _translate("komponentler", "Transistör"))
        self.ozel_tur.setItemText(5, _translate("komponentler", "Kondansatör"))
        self.label.setText(_translate("komponentler", "Listeleme"))
        self.temizle_btn.setText(_translate("komponentler", "    Temizle"))